Formula/xkcd.rb

```diff
@@ -0,0 +1,22 @@
+class Xkcd < Formula
+  desc "Fetch latest, random or any particular xkcd comic right in your terminal"
+  homepage "https://git.hanabi.in/xkcd-go"
+  url "https://git.hanabi.in/repos/xkcd-go.git",
+    tag:      "v2.0.0",
+    revision: "5e68ef5b2e7e6806dd57586e9b7ed4f97f64dba0"
```

Comment:
```suggestion
      tag:      "v2.0.0",
      revision: "5e68ef5b2e7e6806dd57586e9b7ed4f97f64dba0"
```
Let's be consistent with other formulae about aligning the arguments to `url`.